packages/components/src/progress-bar/stories/index.story.tsx

```diff
@@ -29,3 +29,34 @@ const Template: StoryFn< typeof ProgressBar > = ( { ...args } ) => {
 
 export const Default: StoryFn< typeof ProgressBar > = Template.bind( {} );
 Default.args = {};
+
+const withCustomWidthCustomCSS = `
+	.custom-progress-bar {
+		width: 100%;
+	}
+`;
+
+/**
+ * A progress bar with a custom overriden width.
+ *
+ * You can override the default `width` or further customize the behavior
+ * and visuals by passing a custom CSS class via the `cssName` prop.
```

Comment:
```suggestion
 * You can override the default `width` by passing a custom CSS class via the `className` prop.
```

I don't want to recommend _too much_ customization 🫣

Reply:
Fair enough :)